compiler/vm/vm.nim

```diff
@@ -134,18 +134,22 @@ const
 const
   errIllegalConvFromXtoY = "illegal conversion from '$1' to '$2'"
 
+
+proc handleError*(c: TCtx, evt: VmEvent, instLoc = instLoc(-1)) {.inline.} =
+  c.config.handleReport(evt, instLoc)
+
```

Comment:
This seems to be the only reason for the `vmlegacy` import, but it doesn't look like it's used here. Maybe (if not use elsewhere) remove it or move it to `vmlegacy`?

Reply:
removed.